terraform/modules/iam_role_policy/cloudwatch/policy.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents",
+        "logs:DescribeLogStreams",
+        "cloudwatch:ListMetrics",
```

Comment:
Looks like I added the metrics actions last year to use with https://github.com/riemann/riemann-tools/blob/master/tools/riemann-aws/bin/riemann-aws-rds-status. I was thinking we'd use that emitter on more deployments, but so far it's only running on concourse, and hasn't paged us since initial testing (so it's possibly broken or not useful). I'll drop these permissions here and add them to a separate policy that's attached to the atc instances.